build.psm1

```diff
@@ -2114,6 +2116,27 @@ function Start-CrossGen {
     }
 }
 
+# Cleans the PowerShell repo
+# by default everything but the root folder and the Packages folder
+# if you specify -IncludePackages it will clean the Packages folder
+function Start-PSCleanRepo
+{
+    [CmdletBinding()]
+    param(
+        [switch] $IncludePackages
+    )
+    Get-ChildItem $RepoRoot\* -Directory -Exclude 'Packages' | ForEach-Object {
```

Comment:
Resolved

---
In reply to: [77046285](https://github.com/PowerShell/PowerShell/pull/2139#discussion_r77046285) [](ancestors = 77046285)